packages/eslint-config-react-app/index.js

```diff
@@ -272,13 +272,13 @@ module.exports = {
     'jsx-a11y/accessible-emoji': 'warn',
     'jsx-a11y/alt-text': 'warn',
     'jsx-a11y/anchor-has-content': 'warn',
+    'jsx-a11y/anchor-is-valid': 'warn',
```

Comment:
The `anchor-is-valid` rule has quite a bit more reach than the old  `href-no-hash` rule.  It provides three checks:

1. Checks if there is an `href` on an `<a>` element.
2. Checks if `href` attributes contain obviously valid content. 
3. Checks if and `<a>` is obviously used as a button.

The third option seems too restrictive for CRA which renders this setup too restrictive.

Options 1 and 2 I think could be usable. The anchors that would fail for the three rules are listed here https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md#fail

The subchecks can be activated as needed.

To activate options 1 and 2 the config is :
```
"jsx-a11y/anchor-is-valid": [ "warn", {
                    "aspects": [ "noHref", "invalidHref" ]
          }]
```

Just using the `invalidHref` aspect gets closest to what was already in CRA although it will error on anchors with `javascript` in the `href`.

What do you think? Can we activate both aspects?